fineract-provider/build.gradle

```diff
@@ -59,6 +60,8 @@ apply plugin: 'eclipse'
 apply plugin: 'idea'
 apply plugin: 'com.bmuschko.tomcat'
 apply plugin: 'project-report'
+apply plugin: 'java'
```

Comment:
@kangbreder do you happen to remember why you had to add `apply plugin: 'java'` here? Does it work without it as well? I guess it's a bit strange to have a project use BOTH `apply plugin: 'java-library'` AND `apply plugin: 'java'` - it's either or the other.

Reply:
I also think this is not relevant to this swagger changes
@kangbreder can you please verify this.

Reply:
> @kangbreder do you happen to remember why you had to add `apply plugin: 'java'` here? Does it work without it as well? I guess it's a bit strange to have a project use BOTH `apply plugin: 'java-library'` AND `apply plugin: 'java'` - it's either or the other.

@vorburger I have not checking if it works without the plugin. My Ubuntu 18 has failed to boot and i have been trying to correct the problem. Because of that, I have not been able to check that yet